addons/website_slides_survey/static/src/js/public/components/slide_upload_dialog/slide_upload_category.js

```diff
@@ -31,6 +32,15 @@ patch(SlideUploadCategory.prototype, {
             : _t("Select or create a certification");
     },
 
+    _formValidate() {
+        const vals = super._formValidate();
```

Comment:
`const isFormValid = super._formValidate();` ?

Reply:
@awa-odoo 

When adding a certification, the "Title" field is required and turns red if left empty when you try to publish. 
However, if I return ```super``` at the end of the method, the "Title" field won't be marked as required until
a certificate is chosen.

Reply:
I'm not sure what you mean, I was just suggesting to rename the "vals" variable to "isFormValid".

Let me know if I misunderstood your comment.

Reply:
Changes Done, 
I thought you were asking why storing super in the const by the "?" in your commit 😅.